server/test/services/geo/esri/RealEsriClientTest.java

```diff
@@ -73,6 +73,19 @@ public void fetchAddressSuggestionsWithError() throws Exception {
     assertThat(maybeResp.isPresent()).isFalse();
   }
 
+  @Test
+  public void fetchAddressSuggestionsMultipleUrls() throws Exception {
+    helper = new EsriTestHelper(TestType.MULTIPLE_ENDPOINTS);
+    ObjectNode addressJson = Json.newObject();
+    addressJson.put("street", "380 New York St");
+    Optional<JsonNode> maybeResp =
+        helper.getClient().fetchAddressSuggestions(addressJson).toCompletableFuture().get();
+    JsonNode resp = maybeResp.get();
+    ArrayNode candidates = (ArrayNode) resp.get("candidates");
+    assertThat(resp.get("spatialReference").get("wkid").asInt()).isEqualTo(4326);
+    assertThat(candidates).hasSize(8);
```

Comment:
suggestion (blocking): Can you add some comments to this test to explain what it's doing and why 8 candidates are returned?

Reply:
Done